state/workingset.go

```diff
@@ -536,6 +548,18 @@ func (ws *workingSet) handleTsf(tsfs []*action.Transfer) error {
 				}
 				voteeOfSender.VotingWeight.Sub(voteeOfSender.VotingWeight, tx.Amount())
 			}
+
+			if gas < *gasLimit {
```

Comment:
if not, we choose to charge nothing? That seems not to be correct.

Reply:
I think what @lizhefeng said is correct, I do not need to check here, proposal already did this job